authorization_services/topics/enforcer-claim-information-point.adoc

```diff
@@ -0,0 +1,165 @@
+[[_enforcer_claim_information_point]]
+= Claim Information Point
+
+A Claim Information Point (CIP) is responsible for resolving claims and pushing these claims to the {project_name} server in order
+to provide to policies more information about the access context. They can be defined as a configuration option to the `policy-enforcer`
+in order to resolve claims from different sources, such as:
+
+* HTTP Request (parameters, headers, body, headers, etc)
+* External HTTP Service
+* Static values defined in configuration
+* Any other source by implementing the Claim Information Provider SPI
+
+When pushing claims to the {project_name} server, policies can base their decisions not just based on who an user is but taking context
+and contents into account, based upon who, what, why, when, where and which for a given transaction. It is all about Contextual-based Authorization
+and how to use runtime information in order to support fine-grained authorization decisions.
+
```

Comment:
When pushing claims to the {project_name} server, policies can base decisions not only on who an user is but also by taking context and contents into account, based on who, what, why, when, where, and which for a given transaction. It is all about Contextual-based Authorization and how to use runtime information in order to support fine-grained authorization decisions.